Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/SortAdapter.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/DocumentPropertyEditor/SortAdapter.h>
+
+namespace AZ::DocumentPropertyEditor
+{
+    RowSortAdapter::~RowSortAdapter()
+    {
+    }
+
+    void RowSortAdapter::SetSortActive(bool active)
+    {
+        if (m_sortActive != active)
+        {
+            m_sortActive = active;
+            if (active)
+            {
+                // TODO: This method should genenerate patches to/from source once custom patching is implemented
+                HandleReset();
+
```

Comment:
Is this the method where custom patching would be done if the DPE handled move operations?

Reply:
yes, correct.

Reply:
correct, those are notes about what patching to do once move operations are supported by the DPE.